### PR TITLE
Remove sidebar shadow on mobile when closed

### PR DESCRIPTION
**Pull request by ArtMin96**
Before:

After:

### CODE REVIEW

File: packages/admin/resources/views/components/layouts/app/sidebar/index.blade.php [hunk 1]
```diff
@@ -8,7 +8,7 @@
         x-bind:class="$store.sidebar.isOpen ? 'filament-sidebar-open translate-x-0' : '-translate-x-full lg:translate-x-0 rtl:lg:-translate-x-0 rtl:translate-x-full'"
     @endif
     @class([
-        'filament-sidebar fixed inset-y-0 left-0 rtl:left-auto rtl:right-0 z-20 flex flex-col h-screen overflow-hidden shadow-2xl transition-all bg-white lg:border-r rtl:lg:border-r-0 rtl:lg:border-l w-[var(--sidebar-width)] lg:z-0',
+        'filament-sidebar fixed inset-y-0 left-0 rtl:left-auto rtl:right-0 z-20 flex flex-col h-screen overflow-hidden lg:shadow-2xl transition-all bg-white lg:border-r rtl:lg:border-r-0 rtl:lg:border-l w-[var(--sidebar-width)] lg:z-0',
```
Comment (zepfietje) [inline packages/admin/resources/views/components/layouts/app/sidebar/index.blade.php]:
```suggestion
        'filament-sidebar fixed inset-y-0 left-0 z-20 flex h-screen w-[var(--sidebar-width)] flex-col overflow-hidden bg-white transition-all rtl:left-auto rtl:right-0 lg:z-0 lg:border-r lg:shadow-2xl rtl:lg:border-l rtl:lg:border-r-0',
```

**Review by zepfietje — APPROVED** [review]
I've corrected the order of classes, but this PR simply replaces `shadow-2xl` with `lg:shadow-2xl`, @danharrin.

**Comment by zepfietje** [comment]
@ArtMin96, I think we should only remove the shadow when the sidebar is fully closed.
Thoughts?

**Comment by ArtMin96** [comment]
@zepfietje, In that case, lg:shadow-2xl should be written twice.

**Comment by zepfietje** [comment]
How do you mean, @ArtMin96?
Sorry if I don't understand.

**Comment by ArtMin96** [comment]
@zepfietje, I mean in both cases we need to add shadow if I'm not mistaken.

**Comment by zepfietje** [comment]
Yes probably, but that's fine!

**Comment by ArtMin96** [comment]
No problem, I will update the PR

**Pull request merged**